pkg/logs/status/builder.go

```diff
@@ -133,3 +137,10 @@ func (b *Builder) toDictionary(c *config.LogsConfig) map[string]interface{} {
 	}
 	return dictionary
 }
+
+func (b *Builder) getStatusMetrics() map[string]int64 {
+	var metrics = make(map[string]int64, 2)
+	metrics["LogsProcessed"] = b.logsExpVars.Get("LogsProcessed").(*expvar.Int).Value()
```

Comment:
this is too manual, if we had a new metric, it won't be added automatically to the status.

Reply:
Thanks for the review, I'll address the nits right away.

I did not iterate over the logsExpVars to respect the two metrics from the card: https://trello.com/c/hpKBtk6E/1023-agent-status-expose-log-related-metrics-that-are-stored-in-the-expvar-data-to-the-agent-status

I would need Nils to confirm but my guess would be that LogsDecoded makes less sense for customers. Now, I could add all the expvars and just iterate instead.

Reply:
Should we update the metrics though ? I started to work on it a couple of times ago https://github.com/DataDog/datadog-agent/pull/2504/files#diff-eb87589455476ecd4da9932fefcee02dR29

Reply:
Maybe for 6.13 if we get some feedback around those first metrics?
Let's chat why the previous PR did not go through.